config/initializers/core.rb

```diff
@@ -30,6 +30,14 @@
 Core::Registry::Repository[:cms_api] =
   Core::Repository::CMS::CmsApi.new
 
+if Feature.active?(:cms_footer)
+  Core::Registry::Repository[:footer] =
+    Core::Repository::Footer::CMS.new
+else
+  Core::Registry::Repository[:footer] =
+    Core::Repository::Footer::Static.new
+end
+
```

Comment:
About the importance of the feature, I am suspicious to say. But, there is already other place using Cache. It is pretty straight forward to use.